components/org.wso2.carbon.identity.api.user.session/org.wso2.carbon.identity.api.user.session.v1/src/main/java/org/wso2/carbon/identity/rest/api/user/session/v1/impl/UserIdApiServiceImpl.java

```diff
@@ -63,9 +72,33 @@ public Response terminateSessionBySessionId(String userId, String sessionId) {
     @Override
     public Response terminateSessionsByUserId(String userId) {
 
-        Util.validateUserId(SessionManagementServiceHolder.getRealmService(), userId,
-                ContextLoader.getTenantDomainFromContext());
-        sessionManagementService.terminateSessionsByUserId(userId);
-        return Response.noContent().build();
+        try {
+            UserRealm userRealm = CarbonContext.getThreadLocalCarbonContext().getUserRealm();
+            AbstractUserStoreManager userStoreManager = (AbstractUserStoreManager) userRealm.getUserStoreManager();
+
+            if (userStoreManager == null) {
+                log.debug("Returns Null by UserStore Manager");
```

Comment:
check if (log.isDebugEnabled)

Reply:
change log message to "Userstore Manager is null"

Reply:
Added in commit 5a50f558230bca28f5468999859a4ece2b25ded1